test/e2e/config/config-overrides-example.yaml

```diff
@@ -13,8 +13,8 @@ variables:
   VSPHERE_DATASTORE: "WorkloadDatastore"
   VSPHERE_STORAGE_POLICY: "Cluster API vSphere Storage Policy"
   VSPHERE_NETWORK: "network-1"
-  VSPHERE_TEMPLATE: "ubuntu-2404-kube-v1.32.0"
-  FLATCAR_VSPHERE_TEMPLATE: "flatcar-stable-4081.2.0-kube-v1.32.0"
```

Comment:
Name depends on the latest stable version of flatcar when running image-builder.

Currently it is `4152.2.2`:

Output from:

```sh
channel=stable
curl -L -s \
     "https://www.flatcar.org/releases-json/releases-$channel.json" \
    | jq -r 'to_entries[] | "\(.key)"' \
    | grep -v "current" \
    | sort --version-sort \
    | tail -n1
```